matterpy/contrib/rss.py

```diff
@@ -72,10 +72,22 @@ def init(manager, conf):
 
 
 def post_to_text(entry, format_string):
-    title = entry.title
-    url   = entry.link
-    body  = entry.description
+    return_dict = {}
 
-    return format_string.format(
-        title=title, url=url, body=body
-    )
+    for key, value in entry.items():
+
+        if key in format_string:
+            return_dict.update({key : value})
+
+        else:
+            continue
```

Comment:
You don't really need the `continue` statement here, since this is what would happen anyway.

`continue` is needed if you want to abort the iteration because the code run after it is unnecessary. However, in this case the iteration is done anyway.